### DISCUSSION TITLE
Go to market strategy?

### DISCUSSION THREAD

**Discussion by AxGucci** [daveshap/ACE_Framework#5] [General]
Hey guys, I'm pretty new at contributing on Open Source.
I founded 1 successful startup and currently working on my second one.
But even though I understand a few things, I'm no technical, I'm more of a fast implementation founder (work as CEO).

That being said (haha), I think it would be interesting to start thinking about the implementation of the ACE framework into real life, thinking about accessibility and not just the technical side of it.

I'm thinking that a good technology is one that's adopted by its target users, right? Otherwise, it won't make an impact.
When I say "go to market strategy" it's not necessarily to make a profit, but as a concept.

Does this make sense? Are you guys already working on this? 

I get that you're not looking for new collaborators, and I think its a pretty good idea to keep it small, specially regarding de technical team. If I can be of any help, feel free to reach out to me (https://www.linkedin.com/in/axel-gualda/).

TBH i'm afraid to be miss using this, so any feedback is more than welcome and if this shouldn't be posted here, feel free to write it in a comment and I'll delete it.
Thanks!

**Comment by daveshap** [comment]
I'm glad you asked! https://github.com/daveshap/ACE_Framework/blob/main/agile.md

**Reply by AxGucci** [reply]
You need any help with the scrum master position? Not certified but sure AF I can deliver.

**Reply by daveshap** [reply]
The team is having a few meetings, we'll be in touch if we need the help. Since there are several projects that people want to do, we may adopt several teams, in which case a dedicated scrum master would be good.
